src/Foundation/NSUrlSessionHandler.cs

```diff
@@ -280,7 +280,7 @@ protected override void Dispose (bool disposing)
 			}
 		}
 
-		bool allowsCellularAccess;
+		bool allowsCellularAccess = true;
```

Comment:
minor (might be postponed if nothing else needs change) but the `= true;` is not needed anymore (and gets over-written in all, current, cases).

Reply:
I'll merge and will do in a new one.